### PR TITLE
Adding docs on setting up a linux dev environment

### PR DESCRIPTION
**Pull request by etirelli**
Just a short document on what to install on a linux box to compile, test and deploy the operator.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,3 +18,29 @@ If you have a proposed fix for an issue, or an enhancement you would like to mak
...
+
+```bash
+ssh-keygen -t ed25519 -C "<email-registered-on-github-account>"
+# upload public key to github
+
+sudo dnf makecache --refresh
+sudo dnf install -y git-all
+sudo dnf install -y golang
+sudo dnf install -y podman
+sudo dnf install -y docker
```
Comment (zdtsw) [inline CONTRIBUTING.md:34]: docker and podman both?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -18,3 +18,29 @@ If you have a proposed fix for an issue, or an enhancement you would like to mak
 Use a descriptive title, and if it relates to an issue in our tracker please reference which one.
 If the PR is not intended to be merged you should prefix the title with "[WIP]" which indicates it is still Work In Progress.
 PR's description should provide enough information for a reviewer to understand the changes and their relation to the rest of the code.
+
+## Setting up a Linux development environment
```
Comment (zdtsw) [inline CONTRIBUTING.md:22]: maybe add it is only for fedora

File: CONTRIBUTING.md (hunk 1 above)
Comment (etirelli) [reply CONTRIBUTING.md]: removed docker.

**Review by zdtsw — APPROVED** [review]
/lgtm

**Pull request merged**